modules/sql/src/main/scala/EnumDef.scala

```diff
@@ -54,6 +56,7 @@ object EnumDef {
 
     implicit val caseWavelengthNm = at[Wavelength.Nm ](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
     implicit val caseWavelengthUm = at[Wavelength.Um ](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
+    implicit val caseMagnitudeSystem = at[MagnitudeSystem](a => s"gem.sql.MagnitudeSystem.fromString(${a.toString})")
```

Comment:
Here and above you need `gem.enum.MagnitudeSystem` because this is the generated code that goes in core.